hw2/04-background.js

```diff
@@ -1 +1,64 @@
 // Add your code here
+// Function to generate a random color in rgba format with a dimmed alpha value
+function generateRandomColor() {
```

Comment:
Nice, a whole function, however instead of adding alpha as a const, add it as an argument of the function.